draft-ietf-oauth-v2-1.md

```diff
@@ -1880,16 +1783,13 @@ need to be revoked.  Authorization servers MUST ensure the
 integrity of the refresh token value in this case, for example,
 using signatures.
 
-## Refresh Token Response
-
-If valid and authorized, the authorization server issues an access
-token as described in {{access-token-successful-response}}.  If the request failed
-verification or is invalid, the authorization server returns an error
-response as described in {{access-token-error-response}}.
+### Refresh Token Response
```

Comment:
I think we need that first sentence back, it should reference the token response section 4.3 as well.

Reply:
brought the sentence back and changed the reference. ok?